src/ProjectService.Models.Dto/Models/TaskInfo.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+
+namespace LT.DigitalOffice.ProjectService.Models.Dto.Models
+{
+    public class TaskInfo
+    {
+        public Guid Id { get; set; }
+        public Guid TypeId { get; set; }
```

Comment:
here I think instead of all Guids, except Id, the Info model should return, if not, it will generate a bunch of other requests to understand what kind of Guid you sent them

Reply:
Reworked